centreon/src/Core/Dashboard/Application/Exception/DashboardException.php

```diff
@@ -241,4 +241,17 @@ public static function contactGroupIsNotInUserContactGroups(array $contactGroupI
             implode(', ', $contactGroupIds)
         ));
     }
+
+    /**
+     * @param int $dashboardId
+     *
+     * @return self
+     */
+    public static function thumbnailNotFound(int $dashboardId): self
+    {
+        return new self(sprintf(
+            _('The thumbnail provided for dashboard [%s] was not found'),
```

Comment:
```suggestion
            _('The thumbnail provided for dashboard [%d] was not found'),
```

Not problematic to use %s but semantically not correct